packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,70 +1,308 @@
-                    upper_sub_value={localize('{{buy_trades}} buy', { buy_trades: user_data.buy_trades })}
-                    lower_sub_value={localize('{{sell_trades}} sell', { sell_trades: user_data.sell_trades })}
-                />
-                <Cell title={localize('Trades in the last 30 days')} value={user_data.past_month_trades} />
-                <Cell title={localize('30 day completion rate')} value={`${user_data.past_month_completion_rate}%`} />
-                <Cell
-                    title={localize('Average release time')}
-                    value={user_data.average_release_time}
-                    dimension={user_data.average_release_unit}
-                />
+            <div className='my-profile__header'>
+                <div className='my-profile__header-details'>
+                    <div
+                        className='my-profile__header-avatar'
+                        style={{ backgroundColor: generateHexColourFromNickname(nickname) }}
+                    >
+                        {getShortNickname(nickname)}
+                    </div>
+                    <div className='my-profile__header-name'>{nickname}</div>
+                </div>
+                <div className='my-profile__header-verification'>
+                    {has_poi && (
+                        <div>
+                            {localize('ID verified')}
+                            <Icon
+                                className='my-profile__header-verification-icon'
+                                icon='IcCashierVerificationBadge'
+                                size={16}
+                            />
+                        </div>
+                    )}
+                    {has_poa && (
+                        <div className='my-profile__header-verification-status'>
+                            {localize('Address verified')}
+                            <Icon
+                                className='my-profile__header-verification-icon'
+                                icon='IcCashierVerificationBadge'
+                                size={16}
+                            />
+                        </div>
+                    )}
+                </div>
             </div>
+            {has_poi && (
+                <React.Fragment>
+                    <Table>
+                        <Table.Row className='my-profile__stats'>
+                            <div className='my-profile__stats-cell-separator' />
+                            <Table.Cell className='my-profile__stats-cell'>
+                                <div className='my-profile__stats-cell-header'>
+                                    {localize('Daily Limit (Buy/Sell)')}
+                                </div>
+                                <div className='my-profile__stats-cell-info'>
+                                    {`${Math.floor(daily_buy_limit)} / ${Math.floor(daily_sell_limit)}` || '-'}
```

Comment:
The `||` value of `'-'` will never be reached as the left operand always returns a (non-empty) string.